APP-4331 - Visual Testing (#136)
SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles: README.md

|   |– svg/
```

## 🤖 Visual Testing

Automated visual testing uses software to automate the process of comparing visual elements across various screen combinations to uncover visual defects.

> [Creevey](https://github.com/wKich/creevey/) it is an Cross-browser screenshot testing tool for Storybook with fancy UI Runner.

Reviewer: ```suggestion
[Creevey](https://github.com/wKich/creevey/) is a Cross-browser screenshot testing tool for Storybook with a fancy UI Runner.
```
```suggestion
[Creevey](https://github.com/wKich/creevey/) is a Cross-browser screenshot testing tool for Storybook with a fancy UI Runner.
```

- Start storybook and then start Visual Testing Execution.

```bash
yarn start
yarn test
```
And that's it.

> **NOTE:** After changes or new Components, the tests are gonna to fail or need to include in validation.

Reviewer: ```suggestion
**NOTE:** After changes or new Components, the tests will fail or need to be included in validation.
```

In order to update them, you may approve them all in one command:

> ```bash

Reviewer: what if we want to update only one ?

Author: it is the same command.

Reviewer: So if someone adds a new component
And his development breaks another component
We'll take the broken component as a reference thanks to this command?
